aptos-move/aptos-vm/src/aptos_vm.rs

```diff
@@ -661,13 +661,21 @@ impl AptosVM {
                 );
             }
 
-            let info = module_storage
+            let mut info = module_storage
                 .unmetered_get_deserialized_module(module_id.address(), module_id.name())
                 .ok()
                 .flatten()
                 .and_then(|module| get_metadata(&module.metadata))
                 .and_then(|m| m.extract_abort_info(code));
 
+            // If the abort had a message, override the description with the message.
+            if let Some(mut current_info) = current_info {
+                if let Some(info) = info {
+                    current_info.reason_name = info.reason_name;
+                }
+                info = Some(current_info);
```

Comment:
So ignoring error description is the default behaviour? For example, if you have

```rust
/// Some message here
const E_MY_ERROR: u64 = 23;
...

abort 23, std::string::format2("Value too large {} > {}", a, b)
```
then `Some message here` is ignored. Wonder if we should format it as well, cc @wrwg @vgao1996 for thoughts